### PR TITLE
Tempo lab

### PR DESCRIPTION
**Pull request by cloud1230**
The project explores a non-intrusive method which is used to expand the storage and processing capabilities of the existing blockchain platform for time series data without damaging the decentralization feature of the blockchain, so that the blockchain can support more service scenarios, including IoT scenarios.

### CODE REVIEW

File: labs/Tempo.md
Comment (tkuhrt) [inline labs/Tempo.md:17]:
```suggestion
The lab provides the solution to support timepoint data stored based on IoT time series database convergence with blockchain, thus the data can be written, read, aggregated, and analyzed through smart contract. For data aggregation and analysis tasks with a long time line and a large amount of data, parallel computing can be used to accelerate task execution based on the multi-point storage feature of blockchain. Supports latency-sensitive service scenarios (such as traffic governance). Finally, a pure on-link solution can be implemented for IoT time series data, which does not depend on off-link components. Data storage, aggregation, computing, and query can be completely decentralized. It is not bound to a blockchain platform or time series database. You can use plug-ins to flexibly expand the application scenarios (such as Hyperledger Fabric and GaussDB).
```

File: labs/Tempo.md
Comment (tkuhrt) [inline labs/Tempo.md:29]: In order to bring an existing source base to the Hyperledger Foundation, the repository must have [DCO](https://developercertificate.org/) signoff on all commits. Since your existing repository does not have DCO signoff on all commits, you will need to do one of two things:

1. bring your code by squashing all of your commits into a single first commit made against a new lab repo with your DCO sign-off.

2. amend the commit history to include DCO sign-off for each of the commits. The Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). Also, the [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.

Please let us know which option you would like to take for this repository.

Comment (cloud1230) [reply labs/Tempo.md:29]: Thanks tracy for giving suggestions, we are handling the DCO issues.

Comment (cloud1230) [reply labs/Tempo.md:29]: Sorry for reply late. We re-submit and sign all the commits. Please help check. https://github.com/HWBlockchainLab/tempo/tree/dev

Comment (tkuhrt) [reply labs/Tempo.md:29]: All branches and all commits will require DCO sign-off before we can bring this code into the Hyperledger Labs organization. It might be easier to use option 1 and squash all commits into a single commit within the new lab repo. For me, this is the only thing holding up my approval of this as a lab.

Comment (cloud1230) [reply labs/Tempo.md:29]: Thanks for the update, I have deleted the old branch and leave the dco sign-off one. Please help check. 
Thanks again.

**Review by knagware9 — APPROVED** [review]

**Pull request merged**
